cg/cli/demultiplex/demux.py

```diff
@@ -57,10 +60,11 @@ def demultiplex_all(context: CGConfig, flow_cells_directory: click.Path, dry_run
             LOG.warning(f"Can not start demultiplexing for flow cell {flow_cell.id}!")
             continue
 
-        if not flow_cell.validate_sample_sheet():
+        try:
+            sample_sheet_api.validate(flow_cell.sample_sheet_path)
+        except (SampleSheetError, ValidationError):
             LOG.warning(
-                "Malformed sample sheet. "
-                f"Run cg demultiplex sample sheet validate {flow_cell.sample_sheet_path}",
+                f"Malformed sample sheet. Run cg demultiplex samplesheet validate {flow_cell.sample_sheet_path}"

```

Comment:
do we want here to skip this flow cell if the sample sheet fails validation or re-create the sample sheet immediately?